### ISSUE TITLE
USWDS - Disabled styles: High Contrast Components

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds#5198]
## Description

#5063 introduces consistent disabled styles across all components. Viewing these components in high contrast mode exposes further inconsistency across components as well as missing icons & hover states.

![image](https://user-images.githubusercontent.com/25211650/227587800-56c8168f-0113-4c4e-8327-423703ce1a41.png)

_*Note: this green is the disabled system color for high contrast modes and should be expected_

## Steps to reproduce the issue

1. View the [disabled forms pattern](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/cm-disabled-forms-styles/iframe.html?args=&id=patterns-forms--disabled-form-elements&viewMode=story) introduced in #5063 
2. Activate `forced-colors` rendering in your browser's devTools[^1] or through your computers operating system if available
3. Inspect components 

### Additional information 

[^1]:[Instructions: Emulate forced-colors mode ](https://devtoolstips.org/tips/en/emulate-forced-colors/)

**Comment by mahoneycm** [comment]
Work has been completed, waiting for #5063 and #5147 to be merged to continue testing
